Tests/FunctionalTests/NoConfigFunctionalTests.swift

```diff
@@ -36,11 +36,11 @@ class NoConfigFunctionalTests: TestBase {
         resetTestExpectations()
         mockNetworkService.reset()
     }
-    
```

Comment:
@addb do you know why these lines were not caught by swift lint with the previous prs? 
and secondly, can we rather fix these formatting issues in a separate PR in dev?

Reply:
So these are just format warnings so it will not fail the `make lint` command. 

I will open a pr in dev branch as well.